ui/site/src/tvSingle.ts

```diff
@@ -1,7 +1,7 @@
 import * as xhr from 'common/xhr';
-import throttle from 'common/throttle';
+import throttlePromiseDelay from 'common/throttle';
 
-const setZen = throttle(1000, zen =>
+const setZen = throttlePromiseDelay(1000, zen =>
```

Comment:
You will also want to make the second argument an anonymous arrow function, from `1000` to `() => 1000`.

Reply:
Ah good catch, I had alias'd the default export instead of the actual `throttlePromiseDelay` function